dev.ftb.ftb-app.yml

```diff
@@ -0,0 +1,51 @@
+app-id: dev.ftb.ftb-app
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+runtime: org.freedesktop.Sdk
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: ftb.sh
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --env=CHROME_WRAPPER=zypak-wrapper
+  - --filesystem=xdg-download:create # Import/export modpacks and logs
+  - --filesystem=xdg-desktop:create # Import/export modpacks and logs
+  - --filesystem=~/.ftba:create # Home folder for the FTB App
+modules:
+  - name: xrandr
+    sources:
+      - sha256: 7b99edb7970a1365eaf5bcaf552144e4dfc3ccf510c4abc08569849929fb366e
+        type: archive
+        url: https://www.x.org/archive/individual/app/xrandr-1.5.1.tar.gz
+  - name: ftb-app
+    buildsystem: simple
+    build-commands:
+      - chmod +x FTB-App-linux-setup.AppImage
+      - ./FTB-App-linux-setup.AppImage --appimage-extract
+      - cp -r squashfs-root /app/bin/ftb-app
+      - install -D ftb-app.png /app/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -D ${FLATPAK_ID}.desktop -t /app/share/applications
+      - install -D ${FLATPAK_ID}.metainfo.xml -t /app/share/metainfo
+      - install -D ftb.sh /app/bin
+    sources:
+      - type: file
+        dest-filename: FTB-App-linux-setup.AppImage
+        url: https://piston.feed-the-beast.com/app/ftb-app-linux-1.27.3-x86_64.AppImage
+        sha512: 8863dc83389a2cd7049a22a028bc7777497106ee0ad99b28df1b96a10478ae96d1e591a054d4244c217c4e0e09bc8a070bd9ac2f83ba7291ba1c344fcb2d03d6
+        x-checker-data:
+          type: electron-updater
+          url: https://piston.feed-the-beast.com/app/latest-linux.yml
```

Comment:
why not build from source?

Reply:
We've looked into building from source, it seems that we're blocked due to the python application that translates npm packages to required libraries does not support pnpm and we use pnpm as our build tool for our frontend.